packages/react-components/src/components/MessageThread.tsx

```diff
@@ -239,11 +241,11 @@ const generateTextMessageContent = (payload: ChatMessagePayload): JSX.Element =>
 const generateMessageContent = (payload: ChatMessagePayload): JSX.Element => {
```

Comment:
Capitalize to make its own component.

I'm wary of non-component functions returning components. It's very easy to (and, in fact, we have this bug at ToT right now in MessageThread already) end up using a function like this inside a `useMemo`. eslint is unable to detect this, but it cause undefined behaviour at runtime.

Reply:
I did capitalize the other ones, missed this one. Thanks for pointing out.